cmake/build_using_clangw.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Build a given target using clang wrapper.
```

Comment:
We should probably expand a little bit more than this such that someone with no clue about building ELFs on Windows can open this file and not be instantly confused.

Reply:
I have documented the function more thoroughly. Let me know if it reads good.